src/excmds.ts

```diff
@@ -2163,6 +2163,15 @@ export async function tabpush(windowId: number) {
     return activeTabId().then(tabId => browser.tabs.move(tabId, { index: -1, windowId }))
 }
 
+/** Switch to the tab currently playing audio, if any. */
+//#background
+export async function tabaudio() {
+    const tabs = await browser.tabs.query({ currentWindow: true, hidden: false, audible: true })
```

Comment:
I'm wondering if it should operate across all windows

Reply:
I'm not sure. What should happen if the other windows aren't visible, such as when minimized or on a different desktop?

Reply:
We could request focus on the audio window, which seems to work nicely in i3 at least. If the window is on the same desktop, it switches focus, otherwise it sets a needs-attention hint on the desktop it's on.

For minimized windows I guess they could be unminimized. I don't have a way to test it though.

Reply:
I think just setting the tab to active (in whichever window) and letting Firefox worry about the rest is fine : )